Source_Code/Client/ClueLessClient.java

```diff
@@ -362,6 +396,38 @@ else if( statUp instanceof SuggestionWrong)
         {
             System.out.println(((SuggestionWrong) statUp).RefuterName + " was able to refute the suggestion");
         }
+
+        // Notify players about an accusation
+        else if (statUp instanceof AccuseNotification)
+        {
+            // Just recast it because we are going to use so much
+            AccuseNotification accuseNotification = (AccuseNotification) statUp;
+            // Tell them about the accusation
+            System.out.println( accuseNotification );
+
+            // If they (or you) are correct
+            if ( accuseNotification.Correct )
+            {
+                // Tell everyone and end the game
+                System.out.println("That was correct!");
+            }
+
+            // If inccorect
+            else
+            {
+                System.out.println("That was incorrect!");
+                System.out.println(accuseNotification.PlayerName + " was wrong! They are out of the game!");
+            }
+        }
+        // This is sent to a player to let them know they are out of the game.
+        // Do we have to do anything else here?
```

Comment:
We should tell the out-player what the correct answer is (like how you get to see what's actually in the enevelope)

Reply:
Love that. Adding now